bin/crew

```diff
@@ -1268,6 +1268,25 @@ def shrink_dir(dir)
   end
 end
 
+def install_files(prefix, path, dst = File.join( CREW_PREFIX, File.join(prefix, path).delete_prefix(".#{CREW_PREFIX}") ))
```

Comment:
Change `".#{CREW_PREFIX}"` to `"./usr/local"`?  This will be needed if `CREW_PREFIX != '/usr/local'`?

Reply:
I only fixed the code, without commenting on the logic. 🫣

Reply:
@supechicken?

Reply:
@supechicken can you explain the logic here?

Reply:
`prefix` parameter not needed?  should only need src and dst?